### PR TITLE
Unprotected truststores

### PR DESCRIPTION
**Pull request by mstruk**
Add support for truststores that don't have a password set.

### CODE REVIEW

**Comment by mstruk** [comment]
@scholzj This one is ready for review. Whenever you find some time ...

File: oauth-common/src/test/java/io/strimzi/kafka/oauth/common/CommonTests.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        new UnprotectedTruststoreTest().doTest();
+    }
+
+    private void logStart(String msg) {
+        System.out.println();
+        System.out.println();
+        System.out.println("========    "  + msg);
+        System.out.println();
+        System.out.println();
+    }
```
Comment (scholzj) [inline oauth-common/src/test/java/io/strimzi/kafka/oauth/common/CommonTests.java:38]: Is this intentionally left here?

**Review by scholzj — APPROVED** [review]
One nit, but LGTM otherwise. Thanks.

File: oauth-common/src/test/java/io/strimzi/kafka/oauth/common/CommonTests.java (hunk 1 above)
Comment (mstruk) [reply oauth-common/src/test/java/io/strimzi/kafka/oauth/common/CommonTests.java:38]: It is. I use it in other tests as well. It removes any doubts about logging being turned off and allows following progress with minimal output. To reduce the time of starting the dependency containers, many tests are merged into one single JUnit test, and the output helps with following the progress as the 'subtests' within the single JUnit test are started.

Comment (scholzj) [reply oauth-common/src/test/java/io/strimzi/kafka/oauth/common/CommonTests.java:38]: Ok, fine with me in that case.

**Pull request merged**